lms/djangoapps/discussion/templates/discussion/edx_ace/responsenotification/email/body.html

```diff
@@ -19,7 +19,7 @@
             </div>
 
             {% trans "View discussion" as course_cta_text %}
-            {% include "ace_common/edx_ace/common/return_to_course_cta.html" with course_cta_text=course_cta_text course_cta_url=post_link%}
+            {% include "ace_common/edx_ace/common/return_to_course_cta.html" with course_cta_text=course_cta_text|force_escape course_cta_url=post_link%}
```

Comment:
@adeelkhan: Why was this needed?  I would have assumed that since return_to_course_cta.html is a Django Template, it takes care of escaping variables when it goes into the page, and this might just double escape it.  If I am correct, it means that if `course_cta_text` contained an `&`, it might mistakenly become `&amp;amp;` instead of just `&amp;`, and the final page (email?) would display `&amp;`.

Reply:
Done. I actually removed it all together from every instance. I think i was getting an error because of    | force_escape construct previously.